### PR TITLE
Prep for the owl landing in v2

### PR DESCRIPTION
**Pull request by sourishkrout**
Refactoring to get ready for the owl store in Runner v2.

1. Add `Description` into Env Store Panel. Remove `Created`.
2. Refactor how Env Store Monitor is wired up to the Panel for simplification.

This PR requires a `runme` kernel binary release with the latest protobufs (in mainline).

### CODE REVIEW

**Comment by sourishkrout** [comment]
Review, please, @pastuxso. Mainly, you might want to study https://github.com/stateful/vscode-runme/pull/1751/commits/b2b38c0c7e99d1f486ebbce003f888876c7f7853 to understand how to exploit reactive observables and use them without "Events".

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -136,6 +136,14 @@ If you want to run the extension against a specific runme kernel binary, you can
 }
 ```
 
+### Upgrade kernel dependencies
+
+To upgrade the kernel gRPC dependencies, you can run the following command. Be sure to release the `runme` kernel binary before a stable merge/release.
+
+```sh {"background":"false","id":"01JAQYD2R3KN63QZ7EC2T2AB58"}
```
Comment (pastuxso) [inline CONTRIBUTING.md:143]: No name?

**Review by pastuxso — APPROVED** [review]
LGTM 🚀

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -136,6 +136,14 @@ If you want to run the extension against a specific runme kernel binary, you can
...
+npm install -s "@buf/stateful_runme.community_timostamm-protobuf-ts@latest"
```
[the remark this answers is not in the document]
Comment (sourishkrout) [reply CONTRIBUTING.md:144]: Yes, no name is intentional. It's not a frequent-enough task.

**Pull request merged**